### PR TITLE
IDAS.authenticateWithJwt()

### PR DESCRIPTION
**Pull request by rafalgradziel**

### CODE REVIEW

File: WebLibs/api/IDAS.js [hunk 1]
```diff
@@ -40,6 +42,19 @@ export class IDAS {
         }
     }
 
+    async authenticateWithJwt() {
+        if (!authJwtToken) {
+            const url = new URL(apiBaseUrl);
+            url.pathname = "/Session";
+            url.search = "?a=" + appToken;
+            url.search = url.search + "&r=%target%";
```
Comment (Saibamen) [inline WebLibs/api/IDAS.js:50]: There is no `a` and `r` in `SessionController.Index`

Comment (rafalgradziel) [reply WebLibs/api/IDAS.js:50]: I am reworking this code right now.
But SessionController.Index() for sure has these two parameters.

Comment (gdl-reif) [reply WebLibs/api/IDAS.js:50]: Those parameters must be passed by the caller!

Comment (gdl-reif) [reply WebLibs/api/IDAS.js:50]: Same mechanism as current SSO

Comment (Saibamen) [reply WebLibs/api/IDAS.js:50]: Sorry, I was looking at `POST Index`, not `GET Index` method. Everything is OK here

**Pull request merged**